content/webapp/pages/events/[eventId]/index.tsx

```diff
@@ -185,14 +185,11 @@ const EventPage: NextPage<EventProps> = ({
         url: '/events',
         text: 'Events',
       },
-      ...event.series.map(series => {
-        console.log(series);
```

Comment:
Just removed this log (I could see some logs in Kibana, unsure which ones they are but thought I'd remove the ones we don't need).